pages/studentAssessmentExam/studentAssessmentExam.ejs

```diff
@@ -35,7 +36,13 @@
             <input type="hidden" name="__action" value="newInstance">
             <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
             <div class="d-flex justify-content-center">
-              <button id="start-assessment" type="submit" class="btn btn-primary">Start assessment</button>
+              <button id="start-assessment" type="submit" class="btn btn-primary">
+                <% if (assessment.multiple_instance) { %>
+                  New instance 
+                <% } else { %>
+                  Start Assessment
```

Comment:
Looks like the structure of this was changed slightly, the corresponding test case here should be updated to match: https://github.com/PrairieLearn/PrairieLearn/blob/master/tests/testShowClosedAssessment.js#L54.

This is *probably* as simple as just changing that line to:

```javascript
assert.equal(response.$('#start-assessment').text().trim(), 'Start Assessment');
```

Reply:
@nicknytko done! Sorry about the messy commit history :S